src/style.css

```diff
@@ -0,0 +1,28 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars .stars__star {
+  margin-right: 4px;
```

Comment:
write margins after `width` `height`

Reply:
https://codeguide.academy/html-css.html#css-order

here you can read about ordering for properties